src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -747,6 +773,10 @@ impl HummockLevelsExt for Levels {
         }
         delete_sst_ids_set.is_empty()
     }
+
+    fn can_partition_by_vnode(&self) -> bool {
```

Comment:
Nothing to do with the current PR, in another PR I'm going to turn on partitioning in the default cg and the `self.member_table_ids.len() == 1` will fail, any suggestions?

Reply:
Removed.